bundles/org.eclipse.ui.forms/src/org/eclipse/ui/forms/widgets/ExpandableComposite.java

```diff
@@ -887,11 +887,27 @@ public int getExpansionStyle() {
 	/**
 	 * Programmatically changes expanded state.
 	 *
+	 * Equivalent to <code>setExpanded(expanded, false)</code>. @see {@link #setExpanded(boolean, boolean)}.
+	 *
 	 * @param expanded
 	 *            the new expanded state
 	 */
 	public void setExpanded(boolean expanded) {
-		internalSetExpanded(expanded);
+		setExpanded(expanded, false);
+	}
+
+	/**
+	 * Programmatically changes expanded state.
```

Comment:
I know that the documentation for the existing `setExpanded` method is similar, but why need to say "programatically" here? Since a user will never call a method "directly", it will always be used to "programatically" change the state.
I'd propose to remove that (also for the existing method).

Reply:
done